docs/get-started/quick-start.md

```diff
@@ -32,42 +32,51 @@ The following things are mandatory for both local development and production dep
 
 > NOTE: if you're unsure your AWS credentials are configured correctly, you can verify them by running the following command using [AWS CLI](https://aws.amazon.com/cli/): `aws sts get-caller-identity`. If you don't see the user info, take a look at this tutorial to [create an IAM user for programmatic usage](https://www.youtube.com/watch?v=tgb_MRVylWw).
 
-## 1. Install the CLI
+## 1. Create a new project
 
 ```
-npm -g install @webiny/cli
+npx create-webiny-project myProject --template templateName
```

Comment:
I would rather put the final command here. In other words, I'd put the `full` for the `--template` (instead of the `templateName`). That way, users can quickly just c/p the command. And of course, we can then reword the tip below the command snippet accordingly.